### PR TITLE
Add "open in incognito" option for the share to menu

### PR DESCRIPTION
**Pull request by jedireza**
Adds an "Open in Neeva Incognito" option to the share to menu.

### How was this tested?
- [x] I tested this manually

### Manual test cases
Tested in the simulator.

### Screenshots and screen recordings

image

https://user-images.githubusercontent.com/979929/168922479-f2a3e986-5acd-4e51-a129-e32c9432608a.mp4

## Issues addressed
N/A

### CODE REVIEW

File: Extensions/ShareTo/ShareToView.swift [hunk 1]
```diff
@@ -49,6 +49,22 @@ struct ShareToView: View {
                             )
                         }
                         Divider()
+                        Button(action: {
+                            item.url.addingPercentEncoding(
```
Comment (darinwf) [inline Extensions/ShareTo/ShareToView.swift:53]: nit: would probably make sense to refactor this code into a shared helper function to share code between incognito and non-incognito cases.

**Review by darinwf — APPROVED** [review]
Nice! I wonder if there is a way to share the existing incognito.pdf asset instead of forking another copy of it.

File: Extensions/ShareTo/ShareToView.swift (hunk 1 above)
Comment (jedireza) [reply Extensions/ShareTo/ShareToView.swift]: Refactored 👍

**Comment by jedireza** [comment]
> Nice! I wonder if there is a way to share the existing incognito.pdf asset instead of forking another copy of it.

I was able to share the incognito asset but I backed that change out.
- Once I used the shared asset, I thought why not move the `open-in-neeva` asset into the main `Client/Assets/images.xcassets` file too?
- But, I actually like the explicit definition for which assets we use in this `ShareToView`.

Let me know if you have a strong opinion on this.

**Review by darinwf — APPROVED** [review]

**Pull request merged**